src/functional/java/reposense/Entry.java

```diff
@@ -44,17 +44,18 @@ private void verifyAllJson(File expectedDirectory, String actualRelative) {
             if (file.isDirectory()) {
                 verifyAllJson(file, actualRelative);
             } else {
-                if (!file.getName().endsWith(".json")) {
+                if (!file.getName().endsWith(".js")) {
                     continue;
                 }
-                String relativeDirectory = file.getAbsolutePath().split("expected/")[1];
+                String relativeDirectory = file.getAbsolutePath()
+                        .replace("\\", "/").split("expected/")[1];
                 assertJson(file, relativeDirectory, actualRelative);
             }
         }
     }
 
     private void assertJson(File expectedJson, String expectedPosition, String actualRelative) {
-        File actual = new File(actualRelative + expectedPosition);
+        File actual = new File(actualRelative + "/" + expectedPosition);
```

Comment:
Should we use an OS-independent method such as `File.separator` instead of `/`?

Reply:
I'm thinking of switching to [Paths](https://docs.oracle.com/javase/10/docs/api/java/nio/file/Paths.html) once I cleared this up. For those that are going to be printed to json files, we still need to use a fixed standard e.g. `/` though.

Reply:
On second thought, too many File occurrence to fix at the moment. Will change according to suggestion.